nft_id.go

```diff
@@ -113,6 +113,15 @@ func _NftIDFromProtobuf(pb *services.NftID) NftID {
 	}
 }
 
+func _NftIDsFromTokenReferenceProtobuf(rejections []*services.TokenReference) []NftID {
+	nftIDs := make([]NftID, 0)
+
+	for _, v := range rejections {
+		nftIDs = append(nftIDs, _NftIDFromProtobuf(v.GetNft()))
```

Comment:
When all `Rejections` where passed to this function we can assume that there are fts in that `TokenReference` list as well. What would the behaviour be if `v.GetNft()` returns nothing(the case that is a ft).

Reply:
modified